core/src/test/java/io/grpc/internal/RetriableStreamTest.java

```diff
@@ -683,6 +688,48 @@ public void messageAvailable() {
     verify(masterListener).messagesAvailable(messageProducer);
   }
 
+  @Test
+  public void closedWhileDraining() {
+    ClientStream mockStream1 = mock(ClientStream.class);
+    doReturn(mockStream1).when(retriableStreamRecorder).newSubstream();
```

Comment:
Use `doReturn(mockStream1, mockStream2, mockStream3).when(retriableStreamRecorder).newSubstream()` instead?

Reply:
Seems the current version mockito we used does not support vargs.